src/cascadia/TerminalSettingsEditor/TerminalColorConverters.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "TerminalColorConverters.h"
+#include "ColorToBrushConverter.g.cpp"
+#include "ColorToStringConverter.g.cpp"
+#include "BooleanToVisibilityConverter.g.cpp"
+
+using namespace winrt::Microsoft::Terminal::Settings::Model;
+
+namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
+{
+
+    Windows::Foundation::IInspectable ColorToBrushConverter::Convert(Windows::Foundation::IInspectable const& value, Windows::UI::Xaml::Interop::TypeName const& /*targetType*/, Windows::Foundation::IInspectable const& /*parameter*/, hstring const& /*language*/)
+    {
+        const auto color = value.as<Windows::UI::Color>();
+        return Microsoft::Terminal::UI::Converters::ColorToBrush(color);

```

Comment:
fwiw if we're adding a static global converter function, we don't need a converter _object_!

Reply:
Omg, yeah, I hate it. I'd rather directly use `mtu:Converters.ColorToBrush(<param>)` in the XML, but the problem is here:
```XML
<!--  CommonResources.xaml::ColorPreviewTemplate  -->
Fill="{Binding Converter={StaticResource ColorToBrushConverter}}"
```

We don't have a Path set because we want to convert the object itself. Since there isn't a path, there isn't a way to target the object itself (from what I know, at least). `Fill="{Binding mtu:Converters.ColorToBrush}"` _silently_ fails (no rendered Rectangle)! 

Also, if you're curious, we can't use `x:Bind` because `XamlCompiler error WMC1119: This Xaml file must have a code-behind class to use {x:Bind}. See http://go.microsoft.com/fwlink/?LinkID=532920&clcid=0x409 for more information`.

Also also, in case you're extra curious, I wasn't able to figure out how to define the DataType as `IReference<Core::Color>` because of the `IReference` part.

So that's why this part's implemented like this 🫤

Reply:
UGH. Thanks.

Reply:
See https://github.com/microsoft/terminal/pull/17870#issuecomment-2460637117